__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require("../app");
+const db = require("../db/connection");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("GET /api/categories", () => {
+  test("200: returns an array of category objects, each of which has a slug and a description", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body: { categories } }) => {
+        expect(categories).toBeInstanceOf(Array);
+        expect(categories).toHaveLength(4);
+        categories.forEach((category) => {
+          expect(category).toMatchObject(
```

Comment:
You've nested 2 Jest matchers that both do a very similar thing. Consider ().toEqual(expect.objectContaining({...}).

Reply:
I'd rather see this than a strict ().toEqual({..}) though! Well done for avoiding brittle testing 👍